lib/kino/data_table.ex

```diff
@@ -172,9 +175,11 @@ defmodule Kino.DataTable do
   end
 
   @impl true
-  def init({data_rows, data_columns, count, name, sorting_enabled, inspected, formatter}) do
+  def init(
+        {data_rows, data_columns, count, name, sorting_enabled, inspected, formatter, num_rows}
+      ) do
     features = Kino.Utils.truthy_keys(pagination: true, sorting: sorting_enabled)
-    info = %{name: name, features: features}
+    info = %{name: name, features: features, num_rows: num_rows}
```

Comment:
Just so we are consistent with the typespec and never pass `nil` :)

```suggestion
    info = %{name: name, features: features}
    info = if(num_rows, do: Map.put(info, :num_rows, num_rows), else: info)
```